src/Services/Air/transformers/index.js

```diff
@@ -4,6 +4,7 @@ import setPassengersAge from './set-passengers-age';
 import setHasFareBasisFlag from './set-has-farebasis-flag';
 import setGroupsForSegments from './set-groups-for-segments';
 import addMetaPassengersBooking from './add-meta-passengers-booking';
+import decodeExchangeToken from './decode-exchange-token-and-build';
```

Comment:
I mean you use it everywhere as just `decodeExchangeToken`.
I think you could name the file just `decode-exchange-token` then.